source/welcome.svelte

```diff
@@ -0,0 +1,126 @@
+<svelte:options customElement='rgh-welcome' />
+
+<script lang='ts'>
+	import './welcome.css';
```

Comment:
One thing I skipped (for now?) is the use of `<style>` tags because I can't format the CSS in it without also formatting the JS:

- https://github.com/sveltejs/prettier-plugin-svelte/issues/55

JS formatting is taken care of by eslint.

Maybe one day with:

- https://github.com/refined-github/refined-github/pull/7584#issuecomment-2245113330